### PR TITLE
[NUI][Test] Apply home env in launch.json of test projects

### PR DESCRIPTION
**Pull request by everLEEst**
### Description of Change ###

in vscode usage,
every developer should change their HOME directory to launch test before,
so this change apply $HOME enviromental variable insteadly and developer no longer change any files.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by dongsug-song — APPROVED** [review]
great! it works in my local Ubuntu PC.
looks good to me

File: test/Tizen.NUI.StyleGuide/.vscode/launch.json [hunk 1]
```diff
@@ -12,11 +12,11 @@
             "program": "${workspaceFolder}/bin/Debug/netcoreapp3.1/Tizen.NUI.StyleGuide.dll",
             "args": [],
             "env": {
-                "DESKTOP_PREFIX":"/home/{should be fixed}/dali-env/opt",
-                "PATH":"/home/{should be fixed}/dali-env/opt/bin:$PATH",
-                "LD_LIBRARY_PATH":"/home/{should be fixed}/dali-env/opt/lib:$LD_LIBRARY_PATH",
-                "INCLUDEDIR":"/home/{should be fixed}/dali-env/opt/include",
-                "PKG_CONFIG_PATH":"/home/{should be fixed}/dali-env/opt/lib/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig",
+                "DESKTOP_PREFIX":"${env:HOME}dali-env/opt",
```
Comment (jaehyun0cho) [inline test/Tizen.NUI.StyleGuide/.vscode/launch.json:15]: / is missing before dali-env

**Review by jaehyun0cho — CHANGES_REQUESTED** [review]

File: test/Tizen.NUI.StyleGuide/.vscode/launch.json (hunk 1 above)
Comment (everLEEst) [reply test/Tizen.NUI.StyleGuide/.vscode/launch.json]: fixed. thank you.

**Review by jaehyun0cho — APPROVED** [review]
LGTM

**Pull request merged**